packages/aft/lib/src/config/config.dart

```diff
@@ -330,9 +327,23 @@ class PackageInfo
         flavor == PackageFlavor.flutter;
   }
 
+  /// Whether this package is compatible with the current Dart version
+  /// running `aft`.
+  bool get compatibleWithCurrentSdk {
+    final dartSdkConstraint = pubspecInfo.pubspec.environment!['sdk']!;
+    return dartSdkConstraint.allows(
+      Version.parse(Platform.version.split(_whitespace).first),
+    );
+  }
+
+  static final _whitespace = RegExp(r'\s+');
+
   @override
   List<Object?> get props => [name];
 
+  @override
+  String get runtimeTypeName => 'PackageInfo';
```

Comment:
General programming approach question - why do we combine multiple classes into the same file?  As someone unfamiliar with your aft code it's a bit confusing to have these large files to scroll through and also not being able to search by classes by file name that match the class contained.